modules/slacker-api/src/main/java/com/labs2160/slacker/api/Resource.java

```diff
@@ -0,0 +1,6 @@
+package com.labs2160.slacker.api;
+
+public interface Resource extends Configurable {
+
+    void start();
```

Comment:
Add a shutdown() method as well if we are going to say a Resource can be started.